docs/content/distributed-tracing-data-collection/README.md

```diff
@@ -0,0 +1 @@
+# OpenShift Distributed Tracing Data Collection (OpenTelemetry Collector Operator)
```

Comment:
The titles here will appear in the table of contents on the left on the docs page [here](https://www.operate-first.cloud/apps/content/README.html). Currently these are very long: 

![image](https://user-images.githubusercontent.com/10904967/176203479-58d6923a-e3d5-4e36-b0df-9c21ddfe8eee.png)

Can we reduce these down so they occupy one line? 
A more detailed description can be provided in within the README.md which acts as the front page for this section. 

Same goes for `using-distributed-tracing-data-collection.md`

Reply:
yup, i was thinking that, as well. I'll rename them

Reply:
I'm going with `OpenTelemetry Collector` & `Jaeger` since that's what they really are.

Reply:
updated